src/main/kotlin/io/titandata/titan/providers/local/Checkout.kt

```diff
@@ -5,21 +5,44 @@
 package io.titandata.titan.providers.local
 
 import io.titandata.client.apis.CommitsApi
+import io.titandata.client.apis.RepositoriesApi
 import io.titandata.titan.clients.Docker
 import io.titandata.titan.utils.CommandExecutor
 
 class Checkout (
         private val commandExecutor: CommandExecutor = CommandExecutor(),
         private val docker: Docker = Docker(commandExecutor),
-        private val commitsApi: CommitsApi = CommitsApi()
+        private val commitsApi: CommitsApi = CommitsApi(),
+        private val repositoriesApi: RepositoriesApi = RepositoriesApi()
 ) {
-    fun checkout(container: String, guid: String) {
+    fun checkout(container: String, commit: String?, tags: List<String>) {
```

Comment:
Commit is an object in the client SDK, this is why I used guid: String instead of commit: String. We want a GUID string here, not a commit.